rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,345 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-load-balancer-operator"]
+= Tutorial: AWS Load Balancer Operator on ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-load-balancer-operator
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2023-01-03T22:07:08.574151'
+//title: AWS Load Balancer Operator On ROSA
+//aliases: ['/docs/rosa/alb-sts']
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Shaozhen Ding
+//  - Paul Czarkowski
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+
+[TIP]
+====
+Load Balancers created by the AWS Load Balancer Operator (ALBO) cannot be used for {product-title} Routes, and should only be used for individual services or Ingress that does not need the full layer 7 capabilties of a {product-title} route.
+====
+
+link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[AWS Load Balancer (ALB)Controller] is a Kubernetes controller that manages Elastic Load Balancing v2 (ELBv2) for a Kubernetes cluster.
+
+* It satisfies Kubernetes link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress and service resources] by provisioning link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html[Application Load Balancers (ALB)] and
+https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html[Network Load Balancers (NLB)].
+
+Compared with default AWS In Tree Provider, this controller is actively developed with advanced annotations for both ALB and NLB. Some advanced use cases are:
+
+* Using native Kubernetes Ingress with ALB
+* Integrate ALB with web application firewall (WAF)
+* Specify NLB source IP ranges
+* Specify NLB internal IP address
+
+link:https://github.com/openshift/aws-load-balancer-operator[ALBO] is used to used to install, manage and configure an instance of `aws-load-balancer-controller` in a OpenShift cluster.
+
+.Prerequisites
+
+[NOTE]
+====
+ALB requires a multi-AZ cluster, three public subnets split across three AZs in the same VPC as the cluster, and is not suitable for most PrivateLink clusters.
+====
+
+* xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[A multi-AZ ROSA classic cluster] 
```

Comment:
this doc creates a default cluster which is single AZ.    We need to tell them to make sure to add `--multi-az` to the rosa create command.